QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java

```diff
@@ -55,7 +57,7 @@ protected void turn(float degree) throws AutonomousModeOverException,
     protected void collectTote() throws AutonomousModeOverException,
             InterruptedException {
         QuasarHelios.autoLoader.set(true);
-        waitUntil(AutoLoader.done);
+        waitUntil(BooleanMixing.invert((BooleanInput) QuasarHelios.autoLoader));
```

Comment:
Maybe this should be a BooleanInput available on QuasarHelios - `isAutoloaderDone`? Not sure if it'd be better, but it might make this code clearer.